### ISSUE TITLE
Tests fail with openmmtools=0.15.0

### ISSUE THREAD

**Issue by nathanmlim** [MobleyLab/blues#115]
There seems to be an issue with `openmmtools=0.15.0` BLUES tests give:
```blues/simulation.py:176: in generateSimFromStruct
    prop_lambda=prop_lambda
blues/integrators.py:131: in __init__
    self._registered_step_types['H'] = (self._add_alchemical_perturbation_step, False)
../../../anaconda3/envs/hdf5/lib/python3.5/site-packages/simtk/openmm/openmm.py:18023: in <lambda>
    __getattr__ = lambda self, name: _swig_getattr(self, CustomIntegrator, name)
../../../anaconda3/envs/hdf5/lib/python3.5/site-packages/simtk/openmm/openmm.py:74: in _swig_getattr
    return _swig_getattr_nondynamic(self, class_type, name, 0)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <blues.integrators.AlchemicalExternalLangevinIntegrator; proxy of <Swig Object of type 'OpenMM::CustomIntegrator *' at 0x7f459ab59660> >
class_type = <class 'simtk.openmm.openmm.CustomIntegrator'>, name = '_registered_step_types', static = 0

    def _swig_getattr_nondynamic(self, class_type, name, static=1):
        if (name == "thisown"):
            return self.this.own()
        method = class_type.__swig_getmethods__.get(name, None)
        if method:
            return method(self)
        if (not static):
>           return object.__getattr__(self, name)
E           AttributeError: type object 'object' has no attribute '__getattr__'

../../../anaconda3/envs/hdf5/lib/python3.5/site-packages/simtk/openmm/openmm.py:69: AttributeError
```

Reverting to `0.14.0` resolves the error. I'm encountering the error in both `openmm=7.1.1 and 7.2.1`

**Comment by davidlmobley** [comment]
Check recent changes to `openmmtools`?

**Comment by nathanmlim** [comment]
Fixed in PR #154
